src_python/habitat_sim/agent/agent.py

```diff
@@ -12,9 +12,13 @@
 import quaternion as qt
 
 import habitat_sim.errors
-from habitat_sim import bindings as hsim
-from habitat_sim._ext.habitat_sim_bindings import SceneNode
-from habitat_sim.sensors.sensor_suite import SensorSuite
+
+# TODO, trying to figure out how bindings works
+# from habitat_sim import bindings as hsim
+from habitat_sim import scene, sensor
```

Comment:
See @jrreyna `src_python/habitat_sim/bindings/__init__.py`

Reply:
Yeah I eventually found that file 😆. Many of these comments are for my own sake as I make incremental progress.